manipulate_pim_data/rule/general_information_on_rule_format.rst

```diff
@@ -437,14 +437,23 @@ The possible target attribute types are:
  - text
  - textarea
 
+By default a space is added between two source attribute values (not between a source attribute and a source text, or between two source texts). You can avoid this behavior by adding an empty text between the two source attribute values.
+
 **The parameters from and to are required in the format. Depending on the source attribute type, some optional keys can be set:**
 
 +------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| from | List of sets for all attribute types:                                                                                                                                      |
+| from | List of sets for all sources.                                                                                                                                              |
+|      |                                                                                                                                                                            |
+|      | A source must contain exactly one single key among the following ones, plus some extra keys defined further:                                                               |
 |      |                                                                                                                                                                            |
 |      | - field: attribute code.                                                                                                                                                   |
-|      | - locale: locale code for which the value is assigned, for localizable attributes (optional).                                                                              |
-|      | - scope: channel code for which the value is assigned, for scopable attributes (optional).                                                                                 |
+|      | - text: a specific text.                                                                                                                                                   |
+|      | - new_line: no specific value (e.g. ``new_line: ~``). Only available if the target attribute is a text area.                                                               |
```

Comment:
`new_line: ~` is not an example; it's the only way to do, no ?

Reply:
you can use `new_line: true` or `new_line: "I love raclette"` it will work